src/Raven.Server/RavenServer.cs

```diff
@@ -323,7 +326,7 @@ public void RefreshClusterCertificate(object state)
                 }
 
                 // same certificate, but now we need to see if we are need to auto update it
-                var remainingDays = (currentCertificate.Certificate.NotAfter - DateTime.Now).TotalDays;
+                var remainingDays = (currentCertificate.Certificate.NotAfter - Time.GetUtcNow()).TotalDays;
```

Comment:
Need to convert to local time here?

Reply:
Not really... worst case, we'll switch on 29 days and not 30 days